### PR TITLE
Customized control bar for the player

### PR DESCRIPTION
**Pull request by adamjarling**
A partial PR where I finally figured out how VideoJS controlBar options are handled.

image

### CODE REVIEW

**Review by Dananji — APPROVED** [review]
Looks good :+1:

File: src/components/MediaPlayer/VideoJSPlayer.js [hunk 1]
```diff
@@ -29,17 +29,18 @@ function VideoJSPlayer({ isVideo, initStartTime, ...videoJSOptions }) {
...
     });
-
-    // TODO: Wire up the cleanup
-    // Clean up player instance on component unmount
-    //return () => player.dispose();
   }, []);
 
   React.useEffect(() => {
     if (player) {
+      console.log('useEffect() [player]', player);
```
Comment (phuongdh) [inline src/components/MediaPlayer/VideoJSPlayer.js:41]: Looks great, do you still need these `console.log`?

File: src/components/MediaPlayer/VideoJSPlayer.js [hunk 2]
```diff
@@ -68,13 +69,20 @@ function VideoJSPlayer({ isVideo, initStartTime, ...videoJSOptions }) {
...
+      }
+    };
   }, [player]);
 
   React.useEffect(() => {
     if (!player) {
       return;
     }
-
+    console.log('useEffect() [startTime, endTime]', startTime, endTime);
```
Comment (phuongdh) [inline src/components/MediaPlayer/VideoJSPlayer.js:85]: And this one.

Comment (adamjarling) [reply src/components/MediaPlayer/VideoJSPlayer.js:85]: If you don't mind, they're helping me with timing.  I'll get rid of 'em soon.

**Pull request merged**